[2023-03-17T23:17:52.316+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: second_python_dag.first_python_task manual__2023-03-17T23:17:50.356588+00:00 [queued]>
[2023-03-17T23:17:52.334+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: second_python_dag.first_python_task manual__2023-03-17T23:17:50.356588+00:00 [queued]>
[2023-03-17T23:17:52.335+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-03-17T23:17:52.336+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 2
[2023-03-17T23:17:52.337+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-03-17T23:17:52.359+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): first_python_task> on 2023-03-17 23:17:50.356588+00:00
[2023-03-17T23:17:52.364+0000] {standard_task_runner.py:55} INFO - Started process 515 to run task
[2023-03-17T23:17:52.370+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'second_python_dag', 'first_python_task', 'manual__2023-03-17T23:17:50.356588+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/second.py', '--cfg-path', '/tmp/tmpoccmxjk5']
[2023-03-17T23:17:52.371+0000] {standard_task_runner.py:83} INFO - Job 5: Subtask first_python_task
[2023-03-17T23:17:52.569+0000] {task_command.py:388} INFO - Running <TaskInstance: second_python_dag.first_python_task manual__2023-03-17T23:17:50.356588+00:00 [running]> on host 12e59bc4b2cf
[2023-03-17T23:17:52.747+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=second_python_dag
AIRFLOW_CTX_TASK_ID=first_python_task
AIRFLOW_CTX_EXECUTION_DATE=2023-03-17T23:17:50.356588+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-17T23:17:50.356588+00:00
[2023-03-17T23:17:52.755+0000] {logging_mixin.py:137} INFO - Hello I am a Python function called by ***
[2023-03-17T23:17:52.756+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-03-17T23:17:52.775+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=second_python_dag, task_id=first_python_task, execution_date=20230317T231750, start_date=20230317T231752, end_date=20230317T231752
[2023-03-17T23:17:52.862+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-03-17T23:17:52.907+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
